buildspec-unit.yml

```diff
@@ -5,4 +5,4 @@ phases:
     commands:
     - pip install --upgrade pip --quiet
     - pip install tox --quiet
-    - tox -e ${TOX_ENVLIST} -- test/unit
\ No newline at end of file
+    - tox -e ${TOX_ENVLIST} --parallel all -- test/unit
```

Comment:
nit: add a newline to the end of the file

Reply:
Will update in another PR